config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -184,7 +185,6 @@ periodics:
           - --scenario=kubernetes_e2e
           - --
           - --deployment=node
-          - --env=KUBE_SSH_USER=core
```

Comment:
this one is an ubuntu one so doesn't need core

Reply:
Agree